zanata-client-commands/src/main/java/org/zanata/client/commands/pull/PullOptionsImpl.java

```diff
@@ -166,6 +167,16 @@ public boolean isContinueAfterError() {
     }
 
     @Override
+    public int getMinDocPercent() {
+        return this.minDocPercent;
+    }
+
+    @Option(name = "--min-doc-percent", aliases = "-m", usage = "When specified, will only pull down translation files that have translated or better statistics above the specified percentage")
```

Comment:
`--min-doc-translated-percent` is clearer but once we implement the "pull approved only in version requiring review" feature, this will become confusing. Although it's our internal state, but if users don't know the meaning they probably don't know how to use that feature.